src/dataEntities/config.ts

```diff
@@ -1,12 +1,24 @@
 import config from "../config.json";
 
+interface IRateConfig {
+    windowMs: number;
+    max: number;
+    message?: string;
```

Comment:
This isn't used. And I agree that it shouldn't be, I don't think we need to allow configurability of the error message.

Reply:
According to standards, status 429 should include details about what rate limit is in place (in fact I put a custom message in the default config.json.